index_datas/import_datas.py

```diff
@@ -38,7 +38,7 @@ def index_elastic_search(data, elastic_search):
     print('success')
 
 def generate_elements(): 
-    with gzip.open('../../wikidump/enwiki-20210820-abstract.xml.gz', 'rb') as f:
+    with gzip.open('wikidump/enwiki-20210820-abstract.xml.gz', 'rb') as f:
```

Comment:
**[Required]** This will reference a non existent directory `wikidump` inside `index_datas`. It should be `../wikidump/enwiki-20210820-abstract.xml.gz`